src/Omega_h_mesh.cpp

```diff
@@ -1461,4 +1482,18 @@ OMEGA_H_INST(I64)
 OMEGA_H_INST(Real)
 #undef OMEGA_H_INST
 
+#define OMEGA_H_GLOBAL_TAGS_EXPL_INST(T)                                           \
```

Comment:
Perhaps globals are a good candidate for not using ETI, so that the user can really set whatever it wants. OTOH, if these globals are to be written/read to/from mesh, we will need to do a bunch of try/catch with a given set of types, effectively limiting the set of valid types...